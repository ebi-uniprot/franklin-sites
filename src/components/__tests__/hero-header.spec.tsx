import { render } from '@testing-library/react';

import HeroHeader from '../hero-header';

describe('Hero Header component', () => {
  it('should render', () => {
    const { asFragment } = render(
      <HeroHeader title="Title" footer={<span>Footer</span>}>
        <input type="text" />
      </HeroHeader>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
