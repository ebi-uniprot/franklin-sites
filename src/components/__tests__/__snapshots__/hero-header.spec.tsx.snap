// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Header component should render 1`] = `
<DocumentFragment>
  <div
    class="hero-header"
  >
    <div
      class="hero-header__content"
    >
      <h1
        class="x-huge"
      >
        Title
      </h1>
      <input
        type="text"
      />
    </div>
    <div
      class="hero-header__footer"
    >
      <span>
        Footer
      </span>
    </div>
  </div>
</DocumentFragment>
`;
